crates/astria-composer/src/main.rs

```diff
@@ -1,13 +1,14 @@
 use astria_composer::{
-    config,
     telemetry,
     Composer,
+    Config,
 };
+use config::Config as _;
```

Comment:
Very nit: IMO it would nice to have a `fn get::<C: AstriaConfig>` that delegates to the trait so one need not import this. Kind of like `str::parse` which delegates to `FromStr` so one need not import the trait.